timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -155,6 +155,11 @@ services:
         args:
           request: PrepareBackupRequest
         returns: PrepareBackupResponse
+      getFreshTimestamp:
```

Comment:
I think we don't need this, even with the old design. And if we did, we probably want to do it in a different way